sympy/concrete/summations.py

```diff
@@ -946,12 +945,19 @@ def eval_sum(f, limits):
         return eval_sum_direct(f, (i, a, b))
     if isinstance(f, Piecewise):
         return None
+    #If the number of terms is infinite and series is convergent
+    if !(f.is_convergent) and expr.subs(1,a+1)is_positive:
```

Comment:
It doesn't work this way. You should write `if not`

Reply:
yes.I am changing it